.github/workflows/e2e-pr.yml

```diff
@@ -123,18 +126,6 @@ jobs:
                   MRT_TARGET_SETTINGS_FILE_PATH: ${{ vars.MRT_ENV_SETTINGS_E2E_BASE_FILENAME }}
                   MRT_TARGET_VARS_FILE_PATH: ${{ vars.MRT_ENV_VARS_E2E_BASE_FILENAME }}
 
-            # Skip the rest of the workflow for now.
-            - name: Simulating Test Run
-              if: ${{ env.SKIP_WORKFLOW != 'true' }}
-              run: |-
-                  sleep 30
-
-            # Skip the rest of the workflow for now.
-            - name: Skip Workflow
-              if: ${{ env.SKIP_WORKFLOW != 'true' }}
-              run: |-
-                  echo "SKIP_WORKFLOW=true" >> "$GITHUB_ENV"
-
             - name: Get Template Version
```

Comment:
Minor: do we really need this step to get the template version? Since we use create-mobify-app-dev script, it should already use whatever template that's currently in the monorepo.

Reply:
We've had instances in the past where it fallsback to random template versions.
So getting the template version here does 2 things:
- Guarantees we pass in the appropriate version
- We verify the generated app to make sure the template version is what we wanted. So let's leave that in.